src/ERC4973.sol

```diff
@@ -20,12 +21,13 @@ abstract contract ERC4973 is ERC165, IERC721Metadata, IERC4973 {
   constructor(
     string memory name_,
     string memory symbol_
-  ) {
+  ) ERC4973Permit(name_, "1") {
     _name = name_;
     _symbol = symbol_;
   }
 
-  function supportsInterface(bytes4 interfaceId) public view override returns (bool) {
+  function supportsInterface(bytes4 interfaceId) public view override returns (bool)
+  {
```

Comment:
Please don't put the function's start bracket on a new line unless you propose the change consistently to the entire codebase.

Reply:
good catch. need to fix/update my linter